src/components/DocExplorer/DefaultValue.js

```diff
@@ -0,0 +1,31 @@
+/**
+ *  Copyright (c) Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { astFromValue, print } from 'graphql';
+
+export default function DefaultValue({ field }) {
+  const { type, defaultValue } = field;
+  if (defaultValue !== undefined) {
+    return (
+      <span>
+        {' = '}
+        <span className="arg-default-value">
+          {print(astFromValue(defaultValue, type))}
```

Comment:
Does `print` function always print with quotes around the default value? Could we check this with some more test cases, maybe for scalar types at least? (e.g. boolean, int, etc)

Also, the color difference slightly bothers me :D I'm even for using another color hmm... is `arg-default-value` an existing classname?

Reply:
I will add some test cases! This code is similar to the code uthat is currently used to display default values for scalar arguments. The classname that I ised was the same as before. I will take a look to see if there is a more suited color for the default value

Reply:
I ran prettier over the files in the test folders using PR #608.